planner/core/fragment.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package core
+
+import (
+	"github.com/pingcap/tidb/expression"
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/sessionctx"
+	"github.com/pingcap/tidb/util/plancodec"
+	"github.com/pingcap/tipb/go-tipb"
+)
+
+// Fragment is cut from the whole pushed-down plan by pipeline breaker.
+// Communication by pfs are always through shuffling / broadcasting / passing through.
+type Fragment struct {
+	p PhysicalPlan
```

Comment:
rename `p` -> phyPlan?

Reply:
I think single letter naming is a very GOOD convention in go coding. When you see a `p`, you can easily relate 'plan' even though you've never read TiDB code, isn't it ?

Reply:
keep the same  with other files?

Reply:
refer to this https://github.com/pingcap/tidb/blob/e52cb3714e24db63d57cf7879c338b74e59c7227/planner/core/task.go#L796
A simple struct warps a simple field called `p`, clean and elegant.